File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/CustomValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator;

import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator;
import io.github.oleksivio.telegram.bot.core.model.ITelegram;

public interface CustomValidator<T extends ITelegram> extends Validator<T> {
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/CustomValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator

import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator
import io.github.oleksivio.telegram.bot.core.model.ITelegram

interface CustomValidator<T : ITelegram> : Validator<T>
